Documentation/network/kubernetes/ciliumcidrgroup.rst

```diff
@@ -0,0 +1,52 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _CiliumCIDRGroup:
+
+***************************
+CiliumCIDRGroup
+***************************
+
+.. note::
+
+    **Beta Feature:** CiliumCIDRGroup is currently in beta. We welcome your feedback and encourage you to report any issues by filing a GitHub issue.
+
+    For the list of outstanding tasks and known issues, please refer to :gh-issue:`24801`.
+
+CiliumCIDRGroup (CCG) is a feature that allows administrators to group CIDR blocks and reference them in their :ref:`CiliumNetworkPolicy`. Unlike the :ref:`CiliumEndpoint` CRD, which is managed by the Cilium agent, CiliumCIDRGroup is intended to be managed by users. It is particularly useful for grouping and managing external CIDRs.
+
+Below is an example of a ``CiliumCIDRGroup`` object:
```

Comment:
Translators struggle with `below` in this kind of context. Could we use `following` instead?

```suggestion
The following is an example of a ``CiliumCIDRGroup`` object:
```